tools/vscode/refresh_compdb.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+tools/proto_format/proto_format.sh fix
+
+# Setting platform suffix here so the compdb headers won't be overwritten by another bazel run
```

Comment:
As discussion on Slack, probably worth some warnings on what needs to be done when doing API work in this environment.

Reply:
Added in README.md https://github.com/envoyproxy/envoy/pull/11207/files#diff-00554144415ae349ed4a355a828dd494R14